CHANGELOG.md

```diff
@@ -2,38 +2,11 @@
 All notable changes to the Best Buy API will be documented in this file.
 
 
-## [R1.0] - 1900-01-02
+## [R11.1] - 2011.05.01
 ### Added
-- more things!
+Three new **[Products API](https://bestbuyapis.github.io/api-documentation/#products-api)** attributes:
```

Comment:
This line should be part of a list (I know, even when it is a list of just one) and the underlying items should be indented further.

There should also be a line break before this (otherwise markdown gets cranky)